CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## [1.50](https://github.com/brave/brave-ios/releases/tag/v1.50)
+
+### Web3
+
+- Added support for opening ipns:// scheme URL from other apps. (#[7227](https://github.com/brave/brave-ios/issues/7227))
+- Added support for Unstoppable Domains Navigation. (#[7089](https://github.com/brave/brave-ios/issues/7089))
+- Added support for IPFS Navigation in omnibox. (#[7013](https://github.com/brave/brave-ios/issues/7013))
+- Added validation for Brave Wallet Send to disable the "Send" button when address is missing checksum.  (#[7136](https://github.com/brave/brave-ios/issues/7136))
```

Comment:
can be simplified to "Added checksum validation of addresses on the Send screen in Brave Wallet"